src/_locales/en_US/messages.json

```diff
@@ -279,6 +279,10 @@
             }
         }
     },
+    "popup_info_youtube": {
+        "message": "Experiencing issues with YouTube? Are ads playing? It's unlikely that Privacy Badger is the problem. ",
```

Comment:
We should probably take out "Are ads playing?", since the following statement about PB not being the problem is about YT breaking, not about ads. (The statement about ads is that PB is not an ad blocker.)

Reply:
Let's remove the trailing space here and add one where we build the full message. Translators shouldn't have to worry about this stuff if we can help it.

Reply:
Maybe: "Experiencing issues on YouTube? It's probably not Privacy Badger."

Reply:
Makes sense, will do! The conciseness is nice but I'm wondering if "It's probably not Privacy Badger" needs to be more clear about what's not PB. How do you feel about "Privacy Badger is probably not the problem" or "They're probably not caused by Privacy Badger"?

Reply:
"not because of"?

Reply:
perfect!